### PR TITLE
feat: add golang functionality

### PR DESCRIPTION
**Pull request by ndibbern**

### CODE REVIEW

File: generators/deployment/index.js [hunk 1]
```diff
@@ -157,6 +158,15 @@ module.exports = class extends Generator {
 		this.cfIgnoreContent = ['.git/', 'node_modules/', 'test/', 'vcap-local.js'];
 	}
 
+	_configureGo() {
+		// Need a direct github link becasue 'go_buildpack' doesn't have dep support
+		this.manifestConfig.buildpack = 'https://github.com/cloudfoundry/go-buildpack.git#v1.8.25';
+		this.manifestConfig.command = undefined;
```
Comment (jkingoliver) [inline generators/deployment/index.js]: there's no go command to specify here?

File: test/test-kubernetes.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ const scaffolderSampleSwift = scaffolderSample.getJson('SWIFT');
 const scaffolderSampleJava = scaffolderSample.getJson('JAVA');
 const scaffolderSampleSpring = scaffolderSample.getJson('SPRING');
 const scaffolderSamplePython = scaffolderSample.getJson('PYTHON');
+const scaffolderSampleGo = scaffolderSample.getJson('GO');
```
Comment (jkingoliver) [inline test/test-kubernetes.js]: the tests for Go seem light compared to other languages. Can you revisit this and see if there are other meaningful tests we can add?

File: generators/dockertools/resources/go/services.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (jkingoliver) [inline generators/dockertools/resources/go/services.json]: Go won't support these 2 services, right? Is this just a placeholder file? If it's not needed, we should delete it.

File: generators/deployment/index.js (hunk 1 above)
Comment (mblume97) [reply generators/deployment/index.js]: https://docs.cloudfoundry.org/buildpacks/go/index.html#start_command

`If you do not specify a start command in a Procfile, in the manifest, or with the -c flag for cf push, the generated binary will be used as the start command.`

File: test/test-kubernetes.js (hunk 2 above)
Comment (mblume97) [reply test/test-kubernetes.js]: added more!

**Pull request merged**